### PR TITLE
Add Discord icon

### LINKED ISSUE #41: Add link to our developer Discord
Probably just an icon in the top right next to twitter link?

### PR DESCRIPTION
**Pull request by celialewis3**
Fixed #41 -- new discord button!

Also had to update to FontAwesome v5, since there was no Discord in the version the site currently used.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,6 +26,7 @@
     <link rel="stylesheet" href="./style.css">
 
     <!-- import JS -->
+    <script src="https://kit.fontawesome.com/e541009933.js" crossorigin="anonymous"></script>
```
Comment (arirawr) [inline index.html:29]: Let's remove line 25 so we don't import fontawesome twice!

File: index.html [hunk 2]
```diff
@@ -35,9 +36,10 @@
       <header class="header">
         <a href="https://embark-studios.com"><img id="logo" src="./img/logo.png"/></a>
         <div>
-          <a href="https://embark-studios.com" class="fa fa-globe"></a>
+          <a href="https://embark-studios.com" class="fas fa-globe-americas"></a>
           <a href="https://twitter.com/EmbarkStudios" class="fa fa-twitter"></a>
           <a href="https://github.com/EmbarkStudios" class="fa fa-github"></a>
+          <a href="https://discord.gg/ma4NTtB" class="fab fa-discord"></a>
```
Comment (arirawr) [inline index.html:42]: Let's use this link, since it's a permanent one that I know won't be deleted/revoked: `https://discord.gg/8TW9nfF`

**Comment by arirawr** [comment]
This change will also require a CSS change to get the hover effect (yay!)

In `style.css`:
```css
.header .fa:hover {
  transform: scale(1.1);
}
```
should be either changed to affect the fa, fab, and fas classes OR changed to add an additional class (e.g. `.header-icon`) which you can then add to each icon in HTML.

**Comment by celialewis3** [comment]
Cool! Made all 3 changes. 😄

**Comment by arirawr** [comment]
heh, now we've got a stylelint error! you can find it in the "Some checks were not successful" section - I'll let you sleuth it out 🕵️

**Review by arirawr — APPROVED** [review]

**Comment by arirawr** [comment]
Heh, one more thing. Your change to the CSS changes what the CSS does. Instead of `.a .b .c`, it needs to be

```css
.a,
.b, 
.c {
}
```

File: style.css [hunk 3]
```diff
@@ -127,7 +127,10 @@ body > * > * {
   transition: all 0.2s ease-in-out;
 }
 
-.header .fa:hover {
+.header,
+.fa:hover,
+.fab:hover,
+.fas:hover {
```
Comment (arirawr) [inline style.css:133]:
```suggestion
.header .fas:hover {
```

File: style.css [hunk 4]
```diff
@@ -127,7 +127,10 @@ body > * > * {
   transition: all 0.2s ease-in-out;
 }
 
-.header .fa:hover {
+.header,
+.fa:hover,
+.fab:hover,
```
Comment (arirawr) [inline style.css:132]:
```suggestion
.header .fab:hover,
```

File: style.css [hunk 5]
```diff
@@ -127,7 +127,10 @@ body > * > * {
   transition: all 0.2s ease-in-out;
 }
 
-.header .fa:hover {
+.header,
```
Comment (arirawr) [inline style.css:130]:
```suggestion

```

**Comment by celialewis3** [comment]
Jeez, I need more coffee 😆 Thanks, fixed!

File: style.css [hunk 6]
```diff
@@ -127,7 +127,10 @@ body > * > * {
...
+.fa:hover,
```
Comment (arirawr) [inline style.css:131]:
```suggestion
.header .fa:hover,
```

**Pull request merged**